app/src/db/migrations/20230110063945_027-form_components_proactive_help.js

```diff
@@ -0,0 +1,24 @@
+const stamps = require('../stamps');
+
+exports.up = function(knex) {
+  return Promise.resolve()
+    .then(() => knex.schema.createTable('form_components_proactive_help',table=>{
+      table.uuid('id').primary();
+      table.string('componentname').notNullable();
```

Comment:
All of these column names should stay consistent with the rest of the application. They should be camel cased instead of all lower case.